macos_postinstall/src/main.rs

```diff
@@ -0,0 +1,129 @@
+use ini::Ini;
+use itertools::sorted;
+use lazy_static::lazy_static;
+use std::{
+    env,
+    fs::{self, File},
+    io::Write,
+    sync::Mutex,
+};
+
+const LOG_FILE: &str = "/tmp/postinstall_MongoDB_Atlas_SQL_ODBC.log";
+const ODBC_PATH: &str = "/Library/ODBC";
+const INSTALL_ROOT: &str = "/Library/MongoDB/MongoDB Atlas SQL ODBC";
```

Comment:
The name of the driver key is actually "MongoDB Atlas SQL ODBC Driver" I checked the PowerBI and ODBC code.
But please triple check.

Reply:
Tripled checked correct